raspi-config

```diff
@@ -4009,7 +4003,20 @@ do_internationalisation_menu() {
 }
 
 do_advanced_menu() {
-  if gpu_has_mmu ; then
+  if is_pifour ; then
```

Comment:
I don't think gpu_has_mmu is the same as is_pifour - gpu_has_mmu should select both Pi 4 and Pi 5, IIRC - we introduced that test name to keep the existence of Pi 5 secret prior to release. I don't think changing this test is correct.